### PR TITLE
Promote MAPO Images

### PR DESCRIPTION
**Pull request by iamemilio**
MAPO images will be promoted to their own repo in OpenShift 4.10

### CODE REVIEW

**Review by mandre — COMMENTED** [review]
/lgtm

And for the new image mirroring:
/assign smarterclayton

File: core-services/image-mirroring/openshift/mapping_origin_4_10 [hunk 1]
```diff
@@ -177,6 +177,7 @@ registry.ci.openshift.org/origin/4.10:local-storage-operator quay.io/openshift/o
 registry.ci.openshift.org/origin/4.10:local-storage-static-provisioner quay.io/openshift/origin-local-storage-static-provisioner:4.10 quay.io/openshift/origin-local-storage-static-provisioner:4.10.0 quay.io/openshift/origin-local-storage-static-provisioner:latest
 registry.ci.openshift.org/origin/4.10:loki quay.io/openshift/origin-loki:4.10 quay.io/openshift/origin-loki:4.10.0 quay.io/openshift/origin-loki:latest
 registry.ci.openshift.org/origin/4.10:machine-api-operator quay.io/openshift/origin-machine-api-operator:4.10 quay.io/openshift/origin-machine-api-operator:4.10.0 quay.io/openshift/origin-machine-api-operator:latest
+registry.ci.openshift.org/origin/4.10:machine-api-provider-openstack quay.io/openshift/origin-machine-api-provider-openstack:4.10 quay.io/openshift/origin-machine-api-provider-openstack:4.10.0 quay.io/openshift/origin-machine-api-provider-openstack:latest
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_10:180]: I assume other machine api providers will show up using this pattern in the future?  Will it be all providers having their own image?

Comment (mdbooth) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:180]: I think @JoelSpeed or @Fedosin can confirm, but I believe that's consistent with the pattern we're going for.

**Comment by smarterclayton** [comment]
/approve

if the pattern continues (which it seems like it will)

File: core-services/image-mirroring/openshift/mapping_origin_4_10 (hunk 1 above)
Comment (iamemilio) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:180]: Its likely that will be the case

Comment (JoelSpeed) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:180]: Yes, we are intending to move all of the existing machine api providers to this format during the 4.11 timeframe

**Comment by mandre** [comment]
> @iamemilio while reviewing the image names with @smarterclayton we realized that `machine-api-provider-openstack` doesn't align w/ our naming conventions. It should have been `openstack-machine-api-provider` (cloud provider first).
> 
> Can you please rename your image to align w/ this convention? Sorry for the churn caused by catching this late.

Done in https://github.com/openshift/release/pull/25351

**Pull request merged**